implementations.yaml

```
@@ -144,30 +168,22 @@ kems:
         alt_implementations:
           - clean
   hqc:
-    version: 0.0.4
-    notes: |
-      This implementation is not constant-time!
-      This means that it is not secure.
-
-      This crate may remove the ``leaktime`` implementation at any point.
+    version: 0.0.5
```

This can be bumped to `0.1.0`.
```suggestion
    version: 0.1.0
```

---

Done